wum-live-sync-automation/docker/Dockerfile

```diff
@@ -0,0 +1,248 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2016 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# ------------------------------------------------------------------------
+
+FROM ubuntu:trusty
```

Comment:
This Dockerfile should inherit from the WSO2 public docker image instead of bare ubuntu:trusty IMO since this one inherits lot of content from there. I'm trying to see whether we can reduce the duplicate code.. :)

Reply:
Dockerfile cannot be extended, docker images should be extended
dockerfile FROM is used to extend from image not file.
To do that we have to make live docker image public, otherwise, this dockerfile will fail. Thus we cant extend docketfile from the live dockerfile or image

Reply:
As discussed, we will go with live ECS image local copy and extend it with this Dockerfile. BTW, in readme, this will be mentioned for firther reference.